src/main/java/org/broadinstitute/consent/http/db/DataAccessRequestDAO.java

```diff
@@ -313,7 +313,44 @@ void insertDataAccessRequest(
       @Bind("updateDate") Date updateDate,
       @Bind("data") @Json DataAccessRequestData data);
 
+  /**
+   * Create new Progress Report.
+   *
+   * @param parentId       String Parent ID
+   * @param collectionId   Integer DarCollection
+   * @param referenceId    String
+   * @param userId         Integer User
+   * @param createDate     Date Creation Date
+   * @param sortDate       Date Sorting Date
+   * @param submissionDate Date Submission Date
+   * @param updateDate     Date Update Date
+   * @param data           DataAccessRequestData DAR Properties
+   */
+  @RegisterArgumentFactory(JsonArgumentFactory.class)
+  @SqlUpdate(
+      """
+          INSERT INTO data_access_request
+            (parent_id, collection_id, reference_id, user_id, create_date, sort_date, submission_date, update_date, data)
+          VALUES (:parentId, :collectionId, :referenceId, :userId, :createDate, :sortDate,
+            :submissionDate, :updateDate, to_jsonb(:data))
+      """)
+  void insertProgressReport(
+      @Bind("parentId") Integer parentId,
+      @Bind("collectionId") Integer collectionId,
+      @Bind("referenceId") String referenceId,
+      @Bind("userId") Integer userId,
+      @Bind("createDate") Date createDate,
+      @Bind("sortDate") Date sortDate,
+      @Bind("submissionDate") Date submissionDate,
```

Comment:
It looks like all of the uses of insertProgressReport make use of the same date value for all of the date fields and it's always the current time.  Why not just insert the value of now() in the SQL for each of these?  Doing so will have the added benefit of causing someone in the future who wants to set submitted_date to null to have to contemplate the consequences of that decision.

Reply:
This is a pattern we should be using across all inserts

Reply:
> Why not just insert the value of now() in the SQL for each of these?

I agree that the SQL should be setting any fields that we are seeing to `now()` in code, and they should be made non-null.